src/tbb/concurrent_monitor.h

```diff
@@ -23,7 +23,11 @@
 #include "concurrent_monitor_mutex.h"
 #include "semaphore.h"
 
+#if __MAC_OS_X_VERSION_MAX_ALLOWED < 1070
+#include <libkern/OSAtomic.h>
+#else
 #include <atomic>
```

Comment:
This header should be available with C++11.

Reply:
> This header should be available with C++11.

I guess you are right. I will test what are the minimal changes sufficient to unbreak the build once my gcc12 builds are through completion, and drop anything extra. Then whatever left, we can go through to make it compliant with requirements.